README.md

```diff
@@ -1,3 +1,20 @@
 # Pulsar Edit
 
-This is the website repo for Pulsar Edit. It contains our documenation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+This is the website repo for Pulsar Edit. It contains our documentation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+
+
+## Guide
+
+### Recommended Tools:
+- [Toolbox](https://containertoolbx.org/): We recommend using this tool to aide in development as it helps with containerization.
+  - On Linux, Toolbox should be available to install via your native package manager. Alternatively, Toolbox can also be installed by utilizing [Brew](https://brew.sh/).
```

Comment:
`brew` is the name of the CLI tool, but the package manager is called “Homebrew"